intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -1119,8 +1123,17 @@ \subsection{Generic resource requirements}
 
 The job requests 4 GB of RAM memory. As soon as the job tries to use more memory,
 it will be ``killed'' (terminated) by the job scheduler.  There is no
-harm if you \emph{slightly} overestimate the requested memory. \\
- \\
+harm if you \emph{slightly} overestimate the requested memory.
+
+\ifgent
+The default memory on a given cluster is the ``usable memory per node'' divided by
+the ``numbers of cores in a node'' multiplied by the requested processor core(s) (ppn).
+Jobs get the default memory without specifying any memory,
```

Comment:
```suggestion
Jobs will request the default memory without any command line memory specification,
```

Reply:
You should not use any directives for memory in the script file to get the memory, not only command line memory specs. 

command line memory specification or memory directive in the submit script = without any memory specfication